crates/datafusion/src/physical_plan/string_expressions.rs

```diff
@@ -570,6 +570,29 @@ pub fn starts_with<T: StringOffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayR
     Ok(Arc::new(result) as ArrayRef)
 }
 
+/// Returns true if string ends with suffix.
+/// ends_with('alphabet', 'abet') = 't'
+pub fn ends_with<T: StringOffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
```

Comment:
is it possible to move this ends_with to ch_fns module? Because we have added all new CH compatible functions into that.